### DISCUSSION TITLE
[<Axis>] axisLineProps attribute

### DISCUSSION THREAD

**Discussion by valtism** [airbnb/visx#1373] [Ideas]
I was trying to add rounded linecaps on all my lines in a chart, but I realised that although I could do it on an <Axis> for the tick line with `tickLineProps`, there is no corresponding property `axisLineProps` and I am unable to round the line caps for the main axis line.

I think something like an `axisLineProps` attribute would be expected here.

**Comment by williaster** [comment]
Hey @valtism 👋  ah good catch on this. You could still control is via `css` with `.visx-axis-line` as a workaround, but I agree it might be worth adding `axisLineProps` as a catchall for `SVG` `line` element props (some are already there like `stroke` and `strokeDasharray`, but rather than adding more we could add an object prop).

Happy to review a PR if anyone is interested! Can convert this to an issue to track as well.
